docs/install/install-server/configure-server.inc

```diff
@@ -85,10 +103,8 @@ Start the Inmanta server and make sure it is started at boot.
   sudo systemctl enable inmanta-server
   sudo systemctl start inmanta-server
 
-Step 8: Connect to the dashboard
---------------------------------
 
-The server dashboard is now available on port '8888'
+The server dashboard is now available on the port and host configured in step 7.
```

Comment:
To keep things consistent, also use: `sudo systemctl enable --now inmanta-server` here